winui/Shadow/Customization.md

```diff
@@ -128,3 +128,35 @@ shadow.Content = button;
 {% endtabs %}
 
 ![Positioning in WinUI Shadow control](Shadow_images/winui_shadow_offset.png)
+
+## Enable or Disable the shadow
+
+The Shadow control provides support to enable or disable the shadow effect by using the `EnableShadow` property. The default value of the `EnableShadow` property is true.
+
+The below code snippet show how to display the shadow to be false:
```

Comment:
@Priyanka-Vijayasankar The below code snippet shows how to disable the shadow effect:

Reply:
Changed the content as mentioned above.